File: gui/src/app/core/Scripting/Takeout/py/load_args.py
_option_names_map = {
    "init_radius": "inits",
    "num_warmup": "iter_warmup",
    "num_samples": "iter_sampling",
    "num_chains": "chains",
}

if os.path.isfile(os.path.join(HERE, "sampling_opts.json")):
    print("loading sampling_opts.json")
    with open(os.path.join(HERE, "sampling_opts.json")) as f:
        s = json.load(f)
    sampling_opts = {_option_names_map.get(k, k): v for k, v in s.items()}
else:
    sampling_opts = {}

File: gui/src/app/core/Scripting/Takeout/R/load_args.R
.option_names_map = c(
    init_radius="init",
    num_warmup="iter_warmup",
    num_samples="iter_sampling",
    num_chains="chains"
)

sampling_opts <- list()

if (file.exists("./sampling_opts.json")) {
    opts <- jsonlite::fromJSON("./sampling_opts.json")
    for (key in names(opts)) {
        out_key <- key
        if (key %in% names(.option_names_map)) {
            out_key <- .option_names_map[[key]]
        }
        sampling_opts[[out_key]] <- opts[[key]]
    }
}
